packages/ui/packages/Carousel/src/core/slideTrailer/partial.js

```diff
@@ -40,13 +38,12 @@ export const cloneNodeAppendChild = params => {
 
 export const cloneNodeInsertBefore = params => {
   const { perSlide, deepCloneSliderItemsChildren, sliderItems } = params;
-  for (
-    let i = deepCloneSliderItemsChildren.length - perSlide - 1;
-    i < deepCloneSliderItemsChildren.length;
-    i++
-  ) {
-    const cln = deepCloneSliderItemsChildren[i].cloneNode(true);
-    addCloneClass(cln);
-    sliderItems.insertBefore(cln, deepCloneSliderItemsChildren[0]);
-  }
+  const i = deepCloneSliderItemsChildren.length - perSlide - 1;
```

Comment:
I think variable name `i` is not good for `silderThreshold` control

Reply:
done